drivers/storage/virtualbox/virtualbox.go

```diff
@@ -0,0 +1,625 @@
+package virtualbox
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/akutz/gofig"
+	"github.com/akutz/goof"
+	"github.com/appropriate/go-virtualboxclient/virtualboxclient"
```

Comment:
Hi @clintonskitson,

For brevity you may wish to import this as `vbox` or `vboxc`.